fs_test.go

```diff
@@ -442,6 +472,7 @@ func TestFSCompressConcurrent(t *testing.T) {
 		Root:               ".",
 		GenerateIndexPages: true,
 		Compress:           true,
+		CompressBrotli:     true,
 	}
 	h := fs.NewRequestHandler()
 
```

Comment:
`TestFSCompressConcurrent` seems to timeout a lot for this pull request. Can you change the timeout to `time.Second * 2` below?
https://github.com/valyala/fasthttp/blob/763582e8d402b1c5be191867ef27f1fcb21fb633/fs_test.go#L495